python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -189,9 +218,18 @@ def stop(self):
 
     @property
     def stopped(self):
+        """
+        Returns if the flusher has stopped as a boolean?
+        """
         return self._stop.isSet()
 
     def join(self, *args, **kwargs):
+        """
+        Not clear if this is called anywhere
```

Comment:
I didn't write the code I'm just trying to highlight it needs commenting, I'll copy what you've typed there unless you'd prefer to say something else?